String/union.py

```diff
@@ -0,0 +1,18 @@
+def union(list1,list2):
+    le=len(list1)
+    list1[le:le]=list2
+    list4=[]
+    list3=list(dict.fromkeys(list1))
+    for i in range(len(list3)):
+    	list4.append(int(list3[i]))
+    list4.sort()
+    for i in range(len(list4)):
+    	print(list4[i],end=" ")
+    
+        
+M,N=input().split()
```

Comment:
Input function are better written under: 
if __name__ == '__main__':
    M, N = input().split()
And the rest of the code follows.